app/helpers/claims_helper.rb

```diff
@@ -20,7 +20,8 @@ def claim_answers(claim)
       [t("tslr.questions.qts_award_year"), academic_years(claim.qts_award_year), "qts-year"],
       [t("tslr.questions.claim_school"), claim.claim_school_name, "claim-school"],
       [t("tslr.questions.current_school"), claim.current_school_name, "current-school"],
-      [t("tslr.questions.mostly_teaching_eligible_subjects"), (claim.mostly_teaching_eligible_subjects? ? "Yes" : "No"), "subjects-taught"],
+      [t("tslr.questions.subjects_taught"), subject_list(claim.eligible_subjects), "subjects-taught"],
```

Comment:
This is a list of subjects, is it correct to list them with "or"? I think it was correct as it was with "and"?